WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -226,21 +226,19 @@ private bool SanityCheck(IEnumerable<TxOut> expectedOutputs, Transaction unsigne
 
 		private async Task SignTransactionAsync(IEnumerable<AliceClient> aliceClients, Transaction unsignedCoinJoinTransaction, CancellationToken cancellationToken)
 		{
-			async Task<AliceClient?> SignTransactionTask(AliceClient aliceClient)
+			async Task SignTransactionTask(AliceClient aliceClient)
 			{
 				try
 				{
 					await aliceClient.SignTransactionAsync(unsignedCoinJoinTransaction, cancellationToken).ConfigureAwait(false);
-					return aliceClient;
 				}
 				catch (Exception e)
 				{
-					Logger.LogWarning($"Round ({aliceClient.RoundId}), Alice ({{aliceClient.AliceId}}): {nameof(AliceClient.SignTransactionAsync)} failed, reason:'{e}'.");
-					return default;
+					Logger.LogWarning($"Round ({aliceClient.RoundId}), Alice ({aliceClient.AliceId}): Could not sign, reason:'{e}'.");
```

Comment:
Logging `{aliceClient.AliceId}` (the exact string, not the value) is most likely a bug.